packages/mochi-rest/src/modules/pay.ts

```diff
@@ -67,6 +69,8 @@ export class PayModule {
     getSupported: Fetcher<string, Array<Token>>;
   };
 
+  getWhilteListToken: Fetcher<{ symbol?: string }, Array<SimplifiedToken>>;
```

Comment:
hạn chế dùng object nếu input chỉ có 1 value
```suggestion
  getWhilteListToken: Fetcher<string | void, Array<SimplifiedToken>>;
```